navbar-list.html

```diff
@@ -188,20 +188,19 @@ <h2 class="pf-vertical-sub-nav__title">Home and Garden</h2>
     <div
       id="technology-warning"
       aria-hidden="true"
-      aria-labelledby="technology-warning-title"
-      aria-live="polite"
-      role="alertdialog"
       class="pf-c-toast pf-is-warning">
-      <div class="pf-c-toast__icon">
-        <i class="fas fa-home"></i>
-        <span id="technology-warning-title" class="sr-only">Technology Warning:</span>
-      </div>
-      <div id="technology-warning-message" class="pf-c-toast__message">
-        Technology is scary, are you sure you want to proceed?
+      <div role="alert" aria-live="assertive">
```

Comment:
My understanding was that `role="alertdialog"` is expected to take focus from where the user currently is. I don't think this is a match for toast notifications, but instead seems to makes more sense for a modal dialog. Since we don't shift focus, I placed `role="alert"` on the message only, since it seemed pointless to read a button label that they can't click on. We might want to consider providing sr-only text as part of the message that instructs the user on what landmark contains the full alert so that they can access actions related to the message.